mods/ctf/ctf_crafting/init.lua

```diff
@@ -198,15 +180,15 @@ crafting.register_recipe({
 
 crafting.register_recipe({
 	type   = "inv",
-	output = "ctf_traps:dirt 1",
+	output = "ctf_traps:dirt 4",
```

Comment:
```suggestion
	output = "ctf_traps:dirt 5",
```
I recommend to make it 5 because if you put 5 dirt + coal lump, the amount should be the same